integrations/azure/README.md

```diff
@@ -0,0 +1,44 @@
+# Azure ORT starter pipeline
+
+The starter pipeline allows you to easily run an ORT scan on Azure DevOps.
+
+To run the pipeline, simply:
+
+- create a new YAML-based pipeline in your Azure DevOps project based on the pipeline file in this repository
+- run the pipeline with the preferred set of parameters
+
+## Using credentials for my repository
```

Comment:
Bad practice to use "my" in docs, keep things more generic, recommend to change to "Setting up Credentials"